src/lib/node/net.js

```diff
@@ -656,16 +656,36 @@ foam.CLASS({
       class: 'Boolean',
       name: 'followRedirect',
       value: true
+    },
+    {
+      name: 'urlLib',
+      value: require('url')
```

Comment:
This isn't serializable.  I would just keep doing require('url') where needed unless it needs to be saved for performance reasons.